subsys/net/lib/sockets/sockets.c

```diff
@@ -728,12 +757,22 @@ int _impl_zsock_poll(struct zsock_pollfd *fds, int nfds, int timeout)
 
 	pev = poll_events;
 	for (pfd = fds, i = nfds; i--; pfd++) {
+		struct net_context *ctx;
+
 		pfd->revents = 0;
 
 		if (pfd->fd < 0) {
 			continue;
 		}
 
+		ctx = sock_to_net_ctx(pfd->fd);
+
```

Comment:
This empty line should be removed, as we are checking variable after setting it.

Reply:
This is one case is where I disagree (and I prefer to write code with much less empty line than Zephyr in general). In this case, it's not really "checking var after setting it". It's effectively a generalized "switch statement", where first case happens to be checking value of ctx. I.e. with lingo with pattern/expression matching it would be:

~~~
ctx = ...

genswitch {
case ctx == NULL:
...

case pfd->events & ZSOCK_POLLOUT:
...

case pfd->events & ZSOCK_POLLIN:
...
}
~~~

Reply:
Basically you have

```
ctx = sock_to_net_ctx(pfd->fd);
if (ctx == NULL) {
	pfd->revents = ZSOCK_POLLNVAL;
	ret++;
	continue;
}
...
```

so only one if() here which is dealing with ctx variable, the other if() are irrelevant here as they are not dealing with this variable.

Reply:
> This empty line should be removed, as we are checking variable after setting it.
> so only one if() here which is dealing with ctx variable

Ok, fixed.